NEWS.md

```diff
@@ -305,10 +305,10 @@
       unit = "s"
     )
     # Unit: seconds
-    #          expr        min         lq       mean     median         uq        max neval cld
-    #          base 1.37616865 1.39689379 1.86418194 1.54417404 1.91734152 4.27381571    10 a  
-    #     frev_copy 0.52941553 0.59072669 0.76882781 0.65942025 0.72746099 1.35119930    10  b 
-    #  frev_inPlace 0.06466391 0.06529963 0.06621053 0.06580863 0.06682197 0.06955259    10   c
+    #          expr   min    lq  mean median   uq   max neval cld
+    #          base 1.376 1.397 1.864 1.544 1.917 4.274    10 a  
+    #     frev_copy 0.529 0.591 0.769 0.659 0.727 1.351    10  b 
+    #  frev_inPlace 0.064 0.065 0.066 0.066 0.067 0.070    10   c
```

Comment:
this is mixing two naming conventions in single name, I think it is ok to simply `fre_inplace`